applications/openshift/ocp_idp_no_htpasswd/rule.yml

```diff
@@ -14,3 +14,13 @@ severity: medium
 warnings:
     - general: |-
         {{{ openshift_cluster_setting("/apis/config.openshift.io/v1/oauths/cluster") | indent(8) }}}
+
+template:
+    name: yamlfile_value
+    vars:
+        file: cluster.yaml
```

Comment:
Right now compliance-operator expects the file path to match what's in openshift_cluster_setting(), which helps avoid conflicts. Otherwise we would have to verify that no other rule specifies cluster.yaml.

Reply:
Aha. I'll change the template to use `filepath` in case when it is rendered for OCP checks.